ASL/ASLScript.cs

```diff
@@ -122,11 +125,19 @@ public void Update(LiveSplitState state)
             {
                 if (_timer == null)
                     _timer = new TimerModel() { CurrentState = state };
-                TryConnect(state);
+
+                if (_firstUpdate)
+                    TryConnect(state);
+
+                // If the game process was not found in the first update, add a EventManagementWatcher so
+                // we will be notified directly by the OS whenever the target process becomes available
+                if (_game == null && _firstUpdate)
+                    AddWatcher(state);
```

Comment:
There is a race condition here if the process starts in between these two calls.

Try unconditionally add the watcher first and connect after, and have both `TryConnect`/`CallbackTryConnect` give up if already connected to a process.